src/Fixers/ClassKeywordFixer.php

```diff
@@ -72,7 +73,7 @@ public function getName(): string
 
     public function getPriority(): int
     {
-        return 0;
```

Comment:
Maybe hardcoding could be replaced with something like this? https://github.com/blumilksoftware/codestyle/blob/main/src/Fixers/NoCommentFixer.php#L69

Reply:
changed